_posts/2018-05-15-Rust-turns-three.md

```diff
@@ -0,0 +1,109 @@
+of course, [Mozilla](https://blog.rust-lang.org/2017/11/14/Fearless-Concurrency-In-Firefox-Quantum.html);
+it’s also in the top 15 languages on GitHub. As a byproduct, **more and more
+developers are being paid to contribute back to Rust**, many of them full
+time. As of today, Mozilla employees make up only 11% of the official Rust
+teams, and just under half of the total number of people paid to work on
+Rust. (You can read detailed whitepapers about putting Rust into
+production [here](https://www.rust-lang.org/en-US/whitepapers.html).)
+
+Finally, the Rust community continues to work on inclusivity, through outreach
+programs like [Rust Reach](https://blog.rust-lang.org/2018/04/02/Increasing-Rusts-Reach-2018.html) and
+[RustBridge](https://rustbridge.github.io/), as well as
+[structured mentoring](https://blog.rust-lang.org/2017/09/18/impl-future-for-rust.html) and
+investments in [documentation](https://rust-lang-nursery.github.io/rustc-guide/)
+to ease contribution. For 2018, a major goal is to
+[connect and empower Rust’s *global*community](https://blog.rust-lang.org/2018/03/12/roadmap.html),
+which we’re doing both through conference launches in multiple new continents,
+as well as work toward internationalization throughout the project.
+
+# The Product
+
+If you spend much time reading this blog, you’ll know that the major theme of
+our work over the past year has been **productivity**. As we said
+in [last year’s roadmap](https://blog.rust-lang.org/2017/02/06/roadmap.html):
+
+> From tooling to libraries to documentation to the core language, we want to
+> make it easier to get things done with Rust.
+
+This work will culminate in a major release later this year: **Rust 2018
+Edition**. The release will bring together improvements in every area of the
+project, polished into a new “edition” that bundles the changes together with
+updated documentation and onboarding.
+The [roadmap](https://blog.rust-lang.org/2018/03/12/roadmap.html) has some details about
+what to expect.
+
+The components that make up Rust 2018 will be shipped as they become ready on
+the stable compiler. Recent releases include:
+
+- [On-by-default incremental compilation](https://blog.rust-lang.org/2018/02/15/Rust-1.24.html)
+- [A rewritten official book](https://blog.rust-lang.org/2018/05/10/Rust-1.26.html)
+- [Several of the planned language improvements, including `impl Trait` and `match` improvements](https://blog.rust-lang.org/2018/05/10/Rust-1.26.html)
+
+The next couple of releases will
+include [stable SIMD](https://github.com/rust-lang/rust/issues/48556) support,
+procedural macros, custom allocators, and more. The final big features
+— [lifetime system improvements](https://github.com/rust-lang/rfcs/pull/2094)
+and [async/await](https://github.com/rust-lang/rfcs/pull/2394) — should both
+reach feature complete status on nightly within weeks. Vital tools like
+`rustfmt` are also being polished for the new edition with RFCs for finalizing
```

Comment:
We may also want to mention RLS and the IDE work, which has gotten a lot better in the last few weeks, and will also be marching towards a 1.0 release.